Make opening files/folders from the terminal only open in the current instance

Previously they would open in every code-server tab/window.

To test:

1. Run code-server
2. Open code-server
3. Open terminal
4. Open another code-server window
5. Run node ./out/node/entry.js with a file or directory argument

The file or directory should only open from the instance attached to that
terminal.

Index: code-server/lib/vscode/src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts
===================================================================
--- code-server.orig/lib/vscode/src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts
+++ code-server/lib/vscode/src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts
@@ -97,10 +97,14 @@ class RemoteTerminalBackend extends Base
 			}
 			const reqId = e.reqId;
 			const commandId = e.commandId;
+			const terminalId = e.persistentProcessId;
 			if (!allowedCommands.includes(commandId)) {
 				this._remoteTerminalChannel.sendCommandResult(reqId, true, 'Invalid remote cli command: ' + commandId);
 				return;
 			}
+			if (typeof terminalId !== "undefined" && !this._ptys.has(terminalId)) {
+				return
+			}
 			const commandArgs = e.commandArgs.map(arg => revive(arg));
 			try {
 				const result = await this._commandService.executeCommand(e.commandId, ...commandArgs);
